packages/react/slider/package.json

```diff
@@ -24,9 +24,9 @@
     "@radix-ui/react-context": "workspace:*",
     "@radix-ui/react-polymorphic": "workspace:*",
     "@radix-ui/react-primitive": "workspace:*",
-    "@radix-ui/react-use-callback-ref": "workspace:*",
     "@radix-ui/react-use-controllable-state": "workspace:*",
     "@radix-ui/react-use-direction": "workspace:*",
+    "@radix-ui/react-use-layout-effect": "workspace:*",
```

Comment:
We use this package in `collection.tsx` that was moved into `Slider` a while back but we forgot to add the dep.